examples/async_fl/async_aggregator.py

```diff
@@ -200,17 +200,17 @@ def round_completion_handler(self):
 
         # update select participants
         # NOTE: we simulate async, while have to sync every 10 rounds to avoid large division to trace
-        if self.resource_manager.get_task_length() < self.async_buffer_size * 2:
+        if self.resource_manager.get_task_length() < self.async_buffer_size * 5:
```

Comment:
Do we have some insights behind this magic number (i.e., 5)?

Reply:
the reason of choosing a larger magic number is we the start time of resampled client begins at current virtual wall clock time, which has some bubble time with previous sampled clients' start time. So i'd prefer a larger magic number, which makes the bubble time ignorable. Or we can carefully choose the start time of new arrived clients, then this magic number doesn't matter